packages/backend/src/core/discovery/config/DiscoveryConfig.ts

```diff
@@ -32,7 +32,25 @@ export class DiscoveryConfig {
     return this.config.maxDepth ?? 6
   }
 
+  get sharedModules() {
+    return this.config.sharedModules
+      ? Object.values(this.config.sharedModules)
+      : []
+  }
+
   get hash(): Hash256 {
     return hashJson(getDiscoveryConfigEntries(this.config))
   }
+
+  getSharedModule(address: EthereumAddress): string | undefined {
```

Comment:
maybe isSharedModule?

Reply:
I mean this gets `SharedModule | undefined', decided not to create two functions like this.